website/docs/language/meta-arguments/for_each.mdx

```diff
@@ -134,7 +134,7 @@ configuration is applied (such as a unique ID generated by the remote API when
 an object is created).
 
 The `for_each` value must be a map or set with one element per desired
```

Comment:
I talked to the core team, and they actually suggested revising the entire paragraph as follows: 

The `for_each` value must be a map or set with one element per desired resource instance. To use a sequence as the for_each value, you must use an expression that explicitly returns a set value, like the [toset](/language/functions/toset) function. The for_each argument does not implicitly convert lists or tuples to sets to prevent unwanted surprises during conversion.

Thoughts? If you feel comfortable, would you amend your PR with this larger change?

Reply:
sure! ✅